polyglot.css

```diff
@@ -6,3 +6,48 @@
     height: 24px;
     background: rgba(255, 255, 245, 0.8);
 }
+
+.polyglot-lang-select-wfrp4e {
+    flex: 0 0 28px;
+    margin: 6px 6px 6px 2px;
+    box-shadow: inset 0 0 10px #000000;
+    border: 1px solid black;
+    border-image: url(/systems/wfrp4e/ui/chat-controls-border.png) 4 repeat;
+    border-image-width: 4px;
+    border-image-outset: 2px 0px 0px 0px;
+    background: url(/systems/wfrp4e/ui/frames/parchment-texture.png) repeat;
+}
+
+.polyglot-lang-select-wfrp4e label {
+    height: 24px;
+    width: fit-content;
+    min-width: 85px;
+    text-align: center;
+    flex: 0;
+    font-family: CaslonPro;
+    font-weight: 500;
+    font-size: 14px;
+    padding-top: 5px;
+    color: #520000;
+}
+
+.polyglot-lang-select-wfrp4e select {
+    border: none;
+    color: black;
+    font-family: CaslonPro;
+    font-weight: 500;
+    font-size: 14px;
+    padding-top: 3px;
+}
+
+/*
+  These are needed for WFRP to work until the main System is updated with CSS (Verified with DND5E, so should not affect those)
+  Worked with @CatoThe1stElder on the above, so he has the updated WFRP CSS
+*/
+#sidebar .sidebar-tab ol#chat-log {
+    height: calc(100% - 232px);
```

Comment:
Yes I would assume forcing the height would break the Theatre mod.  Hence I would not merge this until the main WFRP system is updated.  But I wanted to get the PR out